### PR TITLE
update mattermost-templated-agreements.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Update the process

### CODE REVIEW

**Comment by natalie-hub** [comment]
@justinegeffen - I'm not able to add reviewers to this PR, so I'm tagging @jamiehurewitz as a reviewer in this comment instead - I hope that's okay.

File: operations/finance/risk-management/mattermost-templated-agreements.md [hunk 1]
```diff
@@ -6,18 +6,18 @@ description: >-
...
 
 ## **Criteria:**
 
-* A firm or an individual with an LLC 
+* An individual with an LLC 
 * Provide ad-hoc services to Mattermost, examples include:  
-  * Project-Based: for a defined period with pre-set project milestones
-  * Ad-hoc services: work that is \(i\) not continuous \(e.g. hours vary month to month\) and \(ii\) typically assigned on an as-needed basis
-
+  * On-going nature: the engagement will exceed 6 months and does not have a fixed end date, e.g. on-going budget approval will not be needed.
```
Comment (jamiehurewitz) [inline operations/finance/risk-management/mattermost-templated-agreements.md]: - We do not typically and should not be procuring consulting services for "ad-hoc" purposes. All procurement of services should have a statement of work detailing the services to be 
- Is there are reason this is time floored at 6 months?
- The contracts should also have a term on them, this is how the original budget is determined through procurement.

File: operations/finance/risk-management/mattermost-templated-agreements.md [hunk 2]
```diff
@@ -6,18 +6,18 @@ description: >-
...
-
+  * On-going nature: the engagement will exceed 6 months and does not have a fixed end date, e.g. on-going budget approval will not be needed. 
+  * System Access requirements: the engagement will require system access currently controlled by IT and/or HR. 
+  
 ## **Process**
 
-1. Confirm you have [budget approval](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval) to engage with a vendor.
+1. Confirm you have budget approval for this with your Team MLT. If you do not have budget approval, see:  [off-cycle budget process](https://handbook.mattermost.com/operations/finance/budget#off-cycle-budget-approval).
 2. The appropriate [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations) should complete the [Mattermost U.S. Consulting Agreement Template](https://app.hellosign.com/s/GnvOJVYW). Reference the [field description guide](https://docs.google.com/document/d/1FKef8xf9N_NTEc0owbAGH4mb0UzU8CJQ5qckGvIl1UM/edit) for help.
```
Comment (jamiehurewitz) [inline operations/finance/risk-management/mattermost-templated-agreements.md]: Can you verify the Template is current and up to date in Hellosign?

File: operations/finance/risk-management/mattermost-templated-agreements.md [hunk 3]
```diff
@@ -6,18 +6,18 @@ description: >-
...
+  * System Access requirements: the engagement will require system access currently controlled by IT and/or HR. 
+  
 ## **Process**
 
-1. Confirm you have [budget approval](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval) to engage with a vendor.
+1. Confirm you have budget approval for this with your Team MLT. If you do not have budget approval, see:  [off-cycle budget process](https://handbook.mattermost.com/operations/finance/budget#off-cycle-budget-approval).
 2. The appropriate [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations) should complete the [Mattermost U.S. Consulting Agreement Template](https://app.hellosign.com/s/GnvOJVYW). Reference the [field description guide](https://docs.google.com/document/d/1FKef8xf9N_NTEc0owbAGH4mb0UzU8CJQ5qckGvIl1UM/edit) for help. 
 3. Upon completion, the CAO will receive the document via HelloSign. Forward that email to [HR](mailto:%20hr@mattermost.com) and include your vendor's signatory contact information \(full name and signature\).
```
Comment (jamiehurewitz) [inline operations/finance/risk-management/mattermost-templated-agreements.md]: If these are vendors, they should run through Procurement and HR is no longer part of the process. Legal took over the consultant agreement process from HR from a vendor/procurement perspective.  However, we shouldn't be having anyone sign an agreement before it has gone through the procurement process, which includes PO approval. If they are agreeing to our standard template, then legal review is not necessary.

File: operations/finance/risk-management/mattermost-templated-agreements.md [hunk 4]
```diff
@@ -6,18 +6,18 @@ description: >-
...
 3. Upon completion, the CAO will receive the document via HelloSign. Forward that email to [HR](mailto:%20hr@mattermost.com) and include your vendor's signatory contact information \(full name and signature\). 
 4. HR will review the document to ensure the [criteria](https://handbook.mattermost.com/operations/finance/risk-management/mattermost-templated-agreements#criteria) are met and will forward to the [Procurement Team](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-4-send-for-internal-signature) for internal [signatures](https://handbook.mattermost.com/operations/operations/company-agreements#who-can-sign-on-behalf-of-the-company) and CC the [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations).
```
Comment (jamiehurewitz) [inline operations/finance/risk-management/mattermost-templated-agreements.md]: See comment above.

**Comment by justinegeffen** [comment]
That's perfect, thanks Natalie!

**Comment by justinegeffen** [comment]
@natalie-hub, just a reminder on the feedback on this PR. Once it's all been addressed, I'll resolve the conflicts so no need to worry about that!

**Comment by jamiehurewitz** [comment]
> @natalie-hub, just a reminder on the feedback on this PR. Once it's all been addressed, I'll resolve the conflicts so no need to worry about that!

**Comment by jamiehurewitz** [comment]
It appears that @ialorro already completed a PR to make changes to this process last week. I don't think these changes need to be made anymore.

**Pull request closed without merging**